docs/sdkx-react-native/notifications-android.mdx

```diff
@@ -0,0 +1,153 @@
+
+To handle notifications from the background we recommend calling `setBackgroundMessageHandler` inside the index.js file where your AppRegistry.registerComponent is called.
+```
+messaging().setBackgroundMessageHandler(async (remoteMessage) => {
+    init();
+    handlePush(remoteMessage.data);
+  });
+  ```
+
+## In-app notifications {#in-app-notifications}
+
+In-app notifications are similar to notifications in the [notification drawer](http://developer.android.com/guide/topics/ui/notifiers/notifications.html). Unlike push notifications, they appear only when your app is running.
+
+These notifications are sent when an agent replies to a customer's issue. Your customers can go straight into the [conversation screen](/sdkx_android/support-tools#conversation-view) when they tap on the notification.
+
+<Centered >
+
+![](/static/books/sdkx_android/notification.png)
+
+</Centered>
+
+<Admonition type="info" title="Note">
+
+If the FCM device token is registered for push notifications, then in-app notifications will be disabled. In-app notifications are disabled to avoid duplicate notifications from both push notifications and in-app notifications.
+
+</Admonition>
+
+## Showing notification count when replies are sent to the user {#showing-notification-count}
+
+To fetch the unread messages count from the server you can call `requestUnreadMessageCount(bool FetchFromServer);` API. This API will return the unread messages count via delegate.
+Based on the value of `FetchFromServer`, the locally stored count will be returned if `FetchFromServer` is `false` else from the server by fetching remotely when `FetchFromServer` is `true`. An example use of this count is to update the badge count to indicate unread messages. Please note that before calling this method, you need to set the listener for Helpshift events by calling the HelpshiftEventListner.
+```javascript
+helpshiftEmitter.addListener('onEventOccurred', (data) => {
+  if (data.eventName == 'receivedUnreadMessageCount') {
+    try {
+      let countData =
+        Platform.OS == 'ios' ? data.eventData : JSON.parse(data.eventData);
+      console.log(countData.count); // show toast to user for count
+    } catch (err) {
+      console.error('Parsing Error', err);
+    }
+  } else {
+    //Your code here.
+  }
+});
+```
+
+
+
+When you call this API, you receive the unread count in your event delegate `onEventOccurred` Listener
+
+- The notification count is fetched via this API from the SDK cache and Helpshift's servers (indicated by the value of `HelpshiftEventDataMessageCountFromCache` in the example above). However, the count from Helpshift’s servers is rate limited and it returns the value only if a subsequent call is made to the API, after the reset timeout, or when the user just closes the chat screen (whichever is earlier). For an active issue, the reset timeout is 1 minute, and 5 minutes for inactive issues.
```

Comment:
Do we have this constant in react code ?
"HelpshiftEventDataMessageCountFromCache"